weblate/settings_docker.py

```diff
@@ -226,6 +226,14 @@
 # Please see the documentation for more details.
 PAGURE_USERNAME, PAGURE_TOKEN, PAGURE_CREDENTIALS = get_env_credentials("PAGURE")
 
+# Bitbucket username and token for sending merge requests.
+# Please see the documentation for more details.
+(
+    BITBUCKETSERVER_USERNAME,
+    BITBUCKETSERVER_TOKEN,
+    BITBUCKETSERVER_CREDENTIALS,
+) = get_env_credentials("BITBUCKETSERVER")
```

Comment:
```suggestion
BITBUCKETSERVER_CREDENTIALS = get_env_credentials("BITBUCKETSERVER")
```

Once https://github.com/WeblateOrg/weblate/pull/8715 is merged.